### ISSUE TITLE
lr and bs config

### ISSUE THREAD

**Issue by YXB-NKU** [showlab/ShowUI#69]
Hello, I would like to ask, during the training process, how significantly do the learning rate and batch size affect the final results? 

I noticed in your paper that you mentioned using 32 V100 GPUs for training, with a learning rate of 0.0001 and an accumulated batch size of 64. 
If I use 8 H100 GPUs for training, would I need to adjust the learning rate and batch size?

**Comment by QinghongLin** [comment]
I think Yes, please slightly decrease the lr to 0.00005 or 0.00001 to enable stable training.
